carsus/io/cmfgen/base.py

```diff
@@ -809,3 +834,92 @@ def _get_levels_lines(self, data):
         self.lines = lines
 
         return
+
+    def _get_collisions(self, data, temperature_grid=None):
+        """
+        Generate the `collisions` DataFrame.
+
+        Parameters
+        ----------
+        data : dict
```

Comment:
Define the inputs

Reply:
Done in 4dac9194e1390d04440482ad75947bb8c44143a5